app/src/components/Cards/methodology-card.tsx

```diff
@@ -70,17 +70,25 @@ const MethodologyCard: FC<MethodologyCardProps> = ({
     >
       <Box w="full" display="flex" justifyContent="space-between">
         <Checkbox
+          variant="subtle"
           disabled={isMethodologyDisabled}
           checked={isSelected}
           onChange={handleRadioChange}
+          css={{
+            borderRadius: "50% !important",
+            "& .chakra-checkbox__control": {
+              borderRadius: "50% !important",
+              background: "base.light",
+            },
+          }}
```

Comment:
Can we add this as default styling for the Checkbox component in the theme/ recipes?